.github/workflows/cypress.yml

```diff
@@ -7,17 +7,60 @@ on:
     branches-ignore:
       - main
 
-# This workflow is made up of one job that calls the reusable workflow in graasp-deploy
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: false
+
 jobs:
-  graasp-deploy-cypress-workflow:
-    # Replace with repository name
-    name: Cypress caller template
-    # Replace 'main' with the hash of a commit, so it points to an specific version of the reusable workflow that is used
-    # Reference reusable workflow file. Using the commit SHA is the safest for stability and security
-    uses: graasp/graasp-deploy/.github/workflows/cypress.yml@v1
-    # Insert required inputs based on repository
-    with:
-      enable-mock-api-test: true
-    # Insert required secrets based on repository with the following format: ${{ secrets.SECRET_NAME }}
-    secrets:
-      api-host-test: ${{ secrets.REACT_APP_API_HOST }}
+  cypress:
+    name: Cypress
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Yarn Install and Cache
+        uses: graasp/graasp-deploy/.github/actions/yarn-install-and-cache@v1
+        with:
+          cypress: true
+
+      # type check
+      - name: Type-check code
+        run: tsc --noEmit
+
+      # eslint check
+      - name: Eslint check code
+        run: yarn lint
+
+      - name: Build App
+        run: yarn build
+        shell: bash
+        env:
+          REACT_APP_API_HOST: ${{ secrets.REACT_APP_API_HOST }}
+
+      #  use the Cypress GitHub Action to run Cypress tests within the chrome browser
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
+        with:
+          install: false
+          # we launch the app in preview mode to avoid issues with hmr websockets from vite polluting the mocks
+          start: yarn preview:test
```

Comment:
You should change this to use the `yarn start`. This preview command "only" exists in vite projects.